Design-Process.md

```diff
@@ -0,0 +1,194 @@
+# Language Design Process
+
+The language design process is the steps that a proposal takes throughout its life, going from an initial seed of an idea, to a championed proposal that is being considered
```

Comment:
OCD question: how is your editor configured? Cause it looks like you have ~100 columns with hard wrap. Generally the markdown settings in the repo push me to different settings. 

It's not so much that I care deeply about this as it is my OCD refused to let me say nothing.